pkg/host/internal/kernel/kernel.go

```diff
@@ -522,6 +522,34 @@ func (k *kernel) InstallRDMA(packageManager string) error {
 	return nil
 }
 
+func (k *kernel) GetRDMASubsystem() (string, error) {
+	log.Log.Info("GetRDMASubsystem(): retrieving RDMA subsystem mode")
+	chrootDefinition := utils.GetChrootExtension()
```

Comment:
We have helper to enter chroot (part of utilsHelper). Do we want to use it here?

Reply:
We'd got the same implementation in all `kernel' methods. Let's do it in a scope of a separate PR